gotpm/cmd/seal_test.go

```diff
@@ -35,13 +35,19 @@ func TestSealPlain(t *testing.T) {
 	ExternalTPM = rwc
 
 	tests := []struct {
-		name   string
-		algo   string
-		pcrArg string
+		name        string
+		algo        string
+		sealPCRs    string
+		certifyPCRs string
 	}{
-		{"RSA", "rsa", ""},
-		{"RSAwithPCRs", "rsa", "7,8"},
-		{"ECCwithPCRs", "ecc", "7"},
+		{"RSA", "rsa", "", ""},
+		{"ECC", "ecc", "", ""},
+		{"RSAWithSealPCR", "rsa", "7", ""},
```

Comment:
Nit: I'd probably call this RSASeal (In general: RSAVerbOptionalverb).

Reply:
RSASeal
RSASealWithPCR
RSACertifyWithPCR
RSASealAndCertifyWithPCR